docker-compose.yml

```diff
@@ -435,16 +385,6 @@ services:
     environment:
       <<: *shared-build-env
 
-  linux-x64x11-sbversion12-evergreen:
```

Comment:
This will break GCB build.  We need to add linux-evergreen service and make change to update_images.py.